app/scripts/lib/tx-verification/tx-verification-middleware.ts

```diff
@@ -1,53 +1,104 @@
 import { hashMessage } from '@ethersproject/hash';
 import { verifyMessage } from '@ethersproject/wallet';
+import type { NetworkController } from '@metamask/network-controller';
+import { rpcErrors } from '@metamask/rpc-errors';
+import { Json, JsonRpcParams, hasProperty, isObject } from '@metamask/utils';
```

Comment:
The `JsonRpc*` types from `@metamask/utils` are broken (they just become `any` in our usage).